test/test_math_utils.cpp

```diff
@@ -64,7 +76,7 @@ SCENARIO("Test that fast logs are invertible and run on device", "[FastMath]") {
       portableFor(
           "try out the fast math", 0, NX, PORTABLE_LAMBDA(const int i) {
             constexpr Real machine_eps = std::numeric_limits<Real>::epsilon();
-            constexpr Real acceptable_err = 100 * machine_eps;
+            constexpr Real acceptable_err = 1000 * machine_eps;
```

Comment:
I guess my same question above applies here

Reply:
see above. It's the `sqrt` in the NQT_2 pow2 function.